### PR TITLE
[12.0][MIG] account_invoice_margin_sale: Migration to 12.0

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa
TT20159

### CODE REVIEW

**Comment by carlosdauden** [comment]
Depends https://github.com/OCA/margin-analysis/pull/66

**Comment by pedrobaeza** [comment]
Please rebase now that the dependency is merged.

**Review by sergio-teruel — COMMENTED** [review]
Waiting for green ci

File: account_invoice_margin_sale/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,8 @@
 {
     "name": "Account Invoice Margin Sale",
     "summary": "Set margin in invoices from sale orders",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
+    'development_status': 'Production/Stable',
```
Comment (pedrobaeza) [inline account_invoice_margin_sale/__manifest__.py]: When promoting to stable, you have to add at least one maintainer

**Review by pedrobaeza — APPROVED** [review]

File: account_invoice_margin_sale/tests/test_account_invoice_margin_sale.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            "uom_po_id": cls.env.ref('uom.product_uom_unit').id,
+            "default_code": "test-margin",
+            "invoice_policy": "order",
+            "list_price": 200.00,
+            "standard_price": 100.00,
+        })
+        cls.product.property_account_receivable_id = cls.account
+        cls.sale_order = cls.env['sale.order'].create({
+            'partner_id': cls.partner.id,
+            'pricelist_id': cls.env.ref('product.list0').id,
```
Comment (sergio-teruel) [inline account_invoice_margin_sale/tests/test_account_invoice_margin_sale.py]: Please use a new pricelist to avoid conflicts with other module tests

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by carlosdauden** [comment]
Changes done

**Comment by pedrobaeza** [comment]
@sergio-teruel do you approve it?

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**